### ISSUE TITLE
Animation did complete hook

### ISSUE THREAD

**Issue by kristian-puccio** [twitter-fabric/velocity-react#20]
Is it possible to get a callback when the animation has completed?

By the way I love this library! Already this has saved me hours.

**Comment by phopkins** [comment]
Happy news! It should be secretly already there. Any `props` you pass to `VelocityComponent` that it doesn't interpret itself get passed in the options hash to Velocity. So, if you put a [`complete`](http://julian.com/research/velocity/#beginAndComplete) property on `VelocityComponent` it should get called.

The [Velocity docs](http://julian.com/research/velocity/) have a list of all the options.

**Comment by phopkins** [comment]
P.S.: Glad to hear you're liking it!

**Comment by kristian-puccio** [comment]
Thanks!

**Comment by nathantalewis** [comment]
I'm not sure if you want this as a separate issue, but it looks like the complete callback doesn't fire on the leave animation of a `VelocityTransitionGroup`.

In the following, `begin` and `process` work happily, but complete is never called.
```jsx
getInitialState() {
  return { maximize: undefined };
},

changeStateToMaximize() {
  this.setState({maximize: true});
},

render() {
  return (<VelocityTransitionGroup
      enter={{ animation: "slideDown" }}
      leave={{ animation: { height: "100%" },
        begin: () => console.log("Beginning..."),
        process: () => console.log("Processing..."),
        complete: () => console.log("Completed.") }}
      runOnMount={true} >
    {this.state.maximize !== undefined ? undefined : (<div key="default" style={{ bottom: 0, height: "200px", left: 0, position: "fixed", width: "100%" }}>
      <div onClick={this.changeStateToMaximize}>
        Inner text
      </div>
    </div>)}
  </VelocityTransitionGroup>);
}
```

**Comment by bsalex** [comment]
@nathantalewis, did you find any workaround?

**Comment by nathantalewis** [comment]
@bsalex, sorry no. We ended up going with react-gsap-enhancer instead. https://github.com/azazdeaz/react-gsap-enhancer

**Comment by bsalex** [comment]
Just in case anyone else has the same problem.
We've ended up using `progress(elements, complete)` callback, checking for `complete === 1`

**Comment by phopkins** [comment]
Put up #53 for this. Anyone who wants `complete` for `VelocityTransitionGroup` care to try it out?

**Comment by py-in-the-sky** [comment]
I just gave the latest of [master](https://github.com/py-in-the-sky/gae-flask-redux-react-starter-kit/commit/1624e9443c5e2caa472bb7f78dd8f01555380b4e#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R66) a [try](https://github.com/py-in-the-sky/gae-flask-redux-react-starter-kit/commit/1624e9443c5e2caa472bb7f78dd8f01555380b4e#diff-e34584f6eff99312dd7c0732eb12af77R12).  Works brilliantly!  Thank you!

When do you plan on your next npm release?

**Comment by phopkins** [comment]
Fixed in 1.1.2, now published.
